src/tests/java/picard/sam/markduplicates/MarkDuplicateWithMissingBarcodeTest.java

```diff
@@ -0,0 +1,35 @@
+package picard.sam.markduplicates;
+
+/**
+ * Created by farjoun on 12/8/15.
+ * <p/>
+ * The purpose of this class is to show that MarkDuplicates gives the same results when run on files that do not have a
+ * molecular barcode tag, even if the code is trying to use the molecular barcode
+ */
+
+abstract public class MarkDuplicateWithMissingBarcodeTest extends MarkDuplicatesTest {
+
+    protected AbstractMarkDuplicatesCommandLineProgramTester getTester() {
+        return new MarkDuplicatesWithMissingBarcodesTester();
+    }
+
+    abstract protected String getTagName();
```

Comment:
rename, since this is the command line argument name, not the name of the tag.  Perhaps `getArgumentName`?  Similarly below, `getArgumentValue`?